pwa-devdocs/src/tutorials/cloud-deploy/index.md

```diff
@@ -2,238 +2,198 @@
+This command runs the build process, which creates a `dist` folder that contains code bundles for your storefront application.
+It also copies over the static assets your application needs from your project into this folder.
 
-Use the Magento Cloud GUI or modify the `variables.env` entry in `.magento.app.yaml` to add any required environment variables.
+## Add your storefront project code
 
-The following table lists the required environment variables for the Venia storefront:
+*In your Cloud project*, create a `pwa` folder and copy into it the content inside your storefront project's `dist` folder.
 
-| Name                                 | Value                                                                                                   |
-| ------------------------------------ | ------------------------------------------------------------------------------------------------------- |
-| `CONFIG__DEFAULT__WEB__UPWARD__PATH` | `/app/pwa/upward.yml` (absolute path to UPWARD YAML configuration)                                      |
-| `NODE_ENV`                           | `production`                                                                                            |
-| `MAGENTO_BACKEND_URL`                | `https://[your-cloud-url-here]`                                                                         |
-| `CHECKOUT_BRAINTREE_TOKEN`           | `<generated token from Braintree>`                                                                      |
-| `MAGENTO_BACKEND_EDITION`            | `EE` (enable Magento Commerce features)                                                                 |
-| `IMAGE_OPTIMIZING_ORIGIN`            | `backend` (use Fastly for image optimization)                                                           |
+```sh
+mkdir pwa && cp -r <path to your storefront project>/dist/* pwa
+```
+
+If you are updating your existing storefront code, you must delete the content inside the `pwa` directory before you copy the new `dist` content to avoid keeping the old bundles the application no longer uses.
 
-## Commit modified files
+## Deploy changes
 
-Commit all changes to the following files:
+At this point in the tutorial, your Cloud project should have changes in the following files and directories:
 
 -   `.magento.app.yaml`
 -   `composer.json`
 -   `composer.lock`
+-   `pwa`
 
-## Push changes
-
-Push your local changes for your deployment and wait for it to complete.
+Use the Git CLI tool to stage, commit, and push these changes to your Cloud project.
 
-## Congratulations
+```sh
+git add . &&
+git commit -m "Added storefront file bundles" &&
+git push origin
+```
 
-You have installed a PWA storefront on the Cloud!
+After you push changes to your Cloud project, the remote build process runs and deploys a live instance of your site to the Magento Commerce Cloud service.
+See the Cloud topic on how to [Deploy your store][] for more details on the deployment process.
```

Comment:
So this becomes tricky when you check the PWA into source, because that bundle is only usable by that branch/environment. If you followed this linked doc and merge integration > staging or staging > prod you'd have the old assets with integration or staging vars hard-coded into the bundle. I've seen partners use git-hooks and `sed` string replacement to solve for this, but that's still kinda fragile.

We can either avoid this topic or call it out that you have to manually merge then re-build the PWA before pushing to staging/prod branches.

Reply:
Addressed in 4ba1c88